<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>75</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_85">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Open Sans Semibold&quot;</string>
       </property>
       <property name="text">
        <string>Calculadora</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_modulos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>941</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60, 121, 182);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_modulo_suma">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>148</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 255);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
     </property>
     <property name="text">
      <string>Sumas</string>
     </property>
     <property name="iconSize">
      <size>
       <width>85</width>
       <height>85</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_modulo_multi">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>144</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 255);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
     </property>
     <property name="text">
      <string>Multiplicaciones</string>
     </property>
     <property name="iconSize">
      <size>
       <width>85</width>
       <height>85</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>931</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_suma">
     <widget class="QFrame" name="frame_31">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>911</width>
         <height>381</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese un numero para sumar: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ln_suma">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>531</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ingresar_suma">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(60, 121, 182);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
        </property>
        <property name="text">
         <string>Ingresar</string>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="resultado_lab_suma">
        <property name="geometry">
         <rect>
          <x>316</x>
          <y>80</y>
          <width>311</width>
          <height>251</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_resultado_suma">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>330</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(60, 121, 182);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
        </property>
        <property name="text">
         <string>Resultado</string>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-1</y>
         <width>911</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>405</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_86">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Open Sans Semibold&quot;</string>
          </property>
          <property name="text">
           <string>Sumador</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>404</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_multi">
     <widget class="QFrame" name="frame_32">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>931</width>
         <height>381</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ingrese un numero para multiplicar: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ln_multi">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>531</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ingresar_multi">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(60, 121, 182);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
        </property>
        <property name="text">
         <string>Ingresar</string>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="resultado_lab_multi">
        <property name="geometry">
         <rect>
          <x>316</x>
          <y>80</y>
          <width>311</width>
          <height>251</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_resultado_multi">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>330</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(60, 121, 182);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
        </property>
        <property name="text">
         <string>Resultado</string>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>921</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>405</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_88">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Open Sans Semibold&quot;</string>
          </property>
          <property name="text">
           <string>Multiplicador</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>404</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>911</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>441</width>
         <height>429</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>140</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Bienvenido a la calculadora </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_inicio">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>148</width>
          <height>61</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Open Sans Semibold&quot;
}

QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(60, 121, 182);
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
font: 75 9pt  &quot;Open Sans Semibold&quot;
}</string>
        </property>
        <property name="text">
         <string>Iniciar</string>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>frame_4</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>frame_modulos</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
